### PR TITLE
setup the project

### PR DESCRIPTION
**Pull request by DjibrilM**
In the following pull request, I implemented 
- web pack 
- CSS linter 
- HTML eslint 
- git workflow 
- folder structure 
- Project's README

### CODE REVIEW

**Review by zainsadaqat — COMMENTED** [review]
# ❌ ❌ ❌Invalid Code Review Request ❌ ❌ ❌
# Hello @DjibrilM  👋  

## Something went wrong 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for this reason:

- ### You have submitted a project, where you followed GitHub flow but according to the project requirement, you're supposed to follow Gitflow. Please follow the correct flow. 👍 
Screenshot 2023-06-28 at 1 41 09 AM

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/template.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>todo</title>
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+<p>something</p>
+</body>
+
+</html>
```
Comment (AlphaNtihinduka) [inline src/template.html]: - [ ] In package.json you stated `index.js` as an a main entry. However, you are missing the index.js file in your `src` folder. I kindly suggest to create index.js file and use Use ES6 modules, with import and export as suggested in the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md#project-requirements).
- [ ] In addition, Kindly follow the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), to write plain HTML markup with minimum styling (just to make the layout),

Your design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @DjibrilM 🙋‍♂️
Good job working on Leaderboard setup. You have done incredibly well. 👏  

TO HIGHLIGHT:
- Have npm run scripts ✔️ 
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- Kindly consider the inline review comments ☺️.

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by Gambit142 — APPROVED** [review]
Hi @DjibrilM 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Set up webpack properly ✔️ 
- UI is identical to that of the wireframe provided ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**